src/Lesson_21/PersonReader.java

```diff
@@ -0,0 +1,55 @@
+package Lesson_21;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+    public class PersonReader {
+        static Scanner sc = new Scanner(System.in);
+        static ArrayList<String> arr = new ArrayList<>();
```

Comment:
It's better to define variables as more abstract types.
Eg. `List<String> arr` instead of `ArrayList<String> arr`